app/src/main/java/at/aau/serg/network/HttpClient.kt

```diff
@@ -7,8 +7,13 @@ import okhttp3.OkHttpClient
 import okhttp3.Request
 import okhttp3.RequestBody.Companion.toRequestBody
 
-class HttpClient(private val baseUrl: String) {
+class HttpClient {
     private var client: OkHttpClient = OkHttpClient()
+    private var baseUrl = "http://10.0.2.2:8081"
```

Comment:
It would be better if we used some environment variable so that we can easily set it and don't have to commit the value.

Reply:
outsourced the value to local.properties